rental.rb

```diff
@@ -0,0 +1,31 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(book, person, date)
+    @book = book
+    @person = person
+    @date = date
+    book.rentals << self
+    person.rentals << self
+  end
+end
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
+
+class Person
+  attr_accessor :name, :age, :rentals
+
```

Comment:
Great job @AyomipoMajek 🤝

- [ ] Kindly ensure there is a method for adding rentals. Kindly follow this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more guidance.

Reply:
Done, thanks